node/src/container_chain_spawner.rs

```diff
@@ -189,6 +191,19 @@ impl ContainerChainSpawner {
                 container_chain_para_id
             );
 
+            if !start_collation {
+                log::info!("This is a syncing container chain, using random ports");
+                // Use random ports to avoid conflicts with the other running container chain
+                let random_ports = [23456, 23457, 23458];
```

Comment:
wont it automatically use random ports even if we dont assign any?, in other words, can we use None here?

Reply:
We maybe can use `[0, 0, 0]`, but the reason I selected the random ports myself is that if we actually use random ports there is a small probability of occupying a port from the collating container. The way it is implemented now that will only happen if this fixed random port is already taken, which can happen in zombienet because all of the collators try to use the same ports. But in production where every collator runs in isolation, I think it is better to use these fixed random ports instead of actually random.